low-level/framework/sspl_reinit

```diff
@@ -54,6 +54,11 @@ usermod -a -G disk sspl-ll
 SUDO_LINE="sspl-ll	ALL = NOPASSWD: /usr/sbin/smartctl, /usr/sbin/mdadm, /usr/bin/mount, /usr/bin/umount, /usr/sbin/swapon, /usr/sbin/swapoff, /usr/sbin/hdparm, /usr/bin/systemctl, /usr/sbin/wbcli, /usr/bin/ipmitool, /usr/bin/systemd-detect-virt, /bin/salt-call, /usr/bin/echo, /usr/bin/facter"
 echo "$SUDO_LINE" | tee /etc/sudoers.d/sspl > /dev/null
 chmod 440 /etc/sudoers.d/sspl
+# Set noop = true in puppet conf file to avoid the sspl auth file getting
+# removed by puppet-agent (Ref: LRL-495)
+PUPPET_CONF="/etc/puppetlabs/puppet/puppet.conf"
```

Comment:
As discussed let's check if the file is present already.

Reply:
@thavanathan, I'm facing issue in pushing the latest changes.
So I created below.

**main** - https://github.com/Seagate/cortx-monitor/pull/154
**cortx** - https://github.com/Seagate/cortx-monitor/pull/155

Reply:
Okay. Closing this one.